wof/wofpy_config.py

```diff
@@ -25,11 +28,13 @@
 
 Options:
   -h --help     Show this screen.
-  -v --version     Show version.
+  -v --version  Show version.
+  --mode=test   deploy mode [default: test]
 """
 
-_ROOT = os.path.abspath(os.path.join(os.pardir, os.path.dirname(__file__)))
-_CONFIG = os.path.join(_ROOT, 'examples', 'flask', 'odm2', 'timeseries')
+_ROOT = os.path.abspath(os.path.dirname(wof.__file__))
+_CONFIG = os.path.join(_ROOT, 'examples', 'production_configs')
+_ODM2_TIMESERIES = os.path.join(_ROOT, 'examples', 'flask', 'odm2', 'timeseries')
```

Comment:
The directories `examples/flask/odm2/timeseries` and ` examples/production_configs` will be copied into the users specified directory. So the tree will be like:

```
usernamedir
    |
    |---- production_configs
    |
    |---- timeseries
```